welcomescreen.py

```diff
@@ -20,92 +20,92 @@
 # Contact information:
 # Utkarsh Tiwari    iamutkarshtiwari@gmail.com
 
+import os
+import sys
+
+import pygame
 import gi
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
-import pygame
-import sys
-from gettext import gettext as _
 
+FPS = 60
+BLACK = (0, 0, 0)
 
-class welcomescreen:
+WELCOMESCREEN_PATH = os.path.join("assets", "welcomescreen.png")
+PLAY_BUTTON_PATH = os.path.join("assets/play.png")
+RULESCREEN_PATH = os.path.join("assets/rules.png")
 
-    def __init__(self):
-        pass
 
-    def run(self, g):
-        black = (0, 0, 0)
-        white = (255, 255, 255)
-        clock = pygame.time.Clock()
-        crashed = False
+class Welcomescreen:
+    def __init__(self, game):
+        self.running = True
+
+        self.welcomescreen = self.load_scaled_image(
+            WELCOMESCREEN_PATH, (490, 768))
+
+        self.play_button = self.load_scaled_image(
+            PLAY_BUTTON_PATH, (120, 120))
+        self.rulescreen = self.load_scaled_image(
+            RULESCREEN_PATH, (490, 768))
 
-        # image load
-        land = pygame.image.load("assets/welcomescreen.png").convert()
-        background = pygame.transform.scale(land, (490, 768))
-        play = pygame.image.load("assets/play.png")
-        play = pygame.transform.scale(play, (120, 120))
-        rules = pygame.image.load("assets/rules.png").convert()
-        rules = pygame.transform.scale(rules, (490, 768))
-        button = pygame.image.load("assets/button.png")
-        font2 = pygame.font.Font("fonts/Arimo.ttf", 25)
-        flag = 1
-        ruleflag = 0
+    def load_scaled_image(self, path, size):
+        width, height = size
+        image = pygame.image.load(path)
+        scaled_image = pygame.transform.scale(
+            image.convert_alpha(), (int(width), int(height)))
+        return scaled_image
 
-        # GAME LOOP BEGINS !!!
-        while not crashed:
-            # Gtk events
+    def run(self, game):
+        clock = pygame.time.Clock()
+        left_click_pressed = False
+        ruleflag = 0  # 0 for welcome screen, 1 for rules screen
+
+        while self.running:
             while Gtk.events_pending():
                 Gtk.main_iteration()
             for event in pygame.event.get():
-                # totaltime+=timer.tick()
                 if event.type == pygame.QUIT:
-                    crashed = True
-                if event.type == pygame.KEYDOWN and (event.key == 276 or event.key == 275):
-                    return
+                    self.running = False
+                if (event.type == pygame.KEYDOWN and
+                   (event.key == pygame.K_LEFT or
+                       event.key == pygame.K_RIGHT)):
+                    if ruleflag == 0:
+                        ruleflag = 1
```

Comment:
This isn't readable, you can check for a `KEYDOWN` event first then check if it's the right or left key that was pressed separately.